### ISSUE TITLE
Beaten to wonder - can't select new production

### ISSUE THREAD

**Issue by MerlinX45** [CQUI-Org/cqui#215]
Egypt beat me to the Hermitage. On the next turn I'm prompted to select new production, but the menu on the right side of the screen does not show up so there is no way to select anything new. This only happens in the city that was producing the wonder. Other city production menus work normally. I tried using the arrow keys to switch cities with the production menu visible. When I got to the city that lost the Hermitage, it showed the previous city's menu and changed production in the other city.

**Comment by chaorace** [comment]
If possible, please include a copy of the affected save

**Comment by MerlinX45** [comment]
Sorry, forgot to add this last night. I have another at turn 423 where production is still running ok. All I have installed are the official DLCs.

[FREDERICK BARBAROSSA 435 1844 AD.zip](https://github.com/chaorace/cqui/files/684755/FREDERICK.BARBAROSSA.435.1844.AD.zip)

**Comment by MerlinX45** [comment]
I played things out a bit differently in this save. Egypt still beat me to the wonder, but I was able to change production. The city still had the wonder listed in the queue, but it replaced it with my new selection. The difference may have been that I had the build menu open in the first game making plans in a different city during the AI turn when the wonder was completed. 
[Turn434.zip](https://github.com/chaorace/cqui/files/685136/Turn434.zip)
